pkg/operator/operator.go

```diff
@@ -104,6 +105,14 @@ func New(config *rest.Config, version, namespace, namespaceSelector, configMapNa
 		UpdateFunc: func(_, newObj interface{}) { o.handleEvent(newObj) },
 	})
 
+	o.openshiftConfigCmapInf = cache.NewSharedIndexInformer(
+		o.client.ConfigMapListWatchForNamespace("openshift-config"),
```

Comment:
Why are we watching for configmap changes in this namespace?

Reply:
Oh, i brain fart. didn't see that the informer was for configmap.

is there any other informer that needs setup so that the operator is responding to changes to this secret or no sync was setup for previous secret location too.